lib/knackhq/client.rb

```diff
@@ -38,13 +38,13 @@ def fields(object)
     end
 
     def records(object, options = {})
-      rows_per_page = options[:rows_per_page] || 25
-      page_number = options[:page_number] || 1
+      rows_per_page = options.delete(:rows_per_page) || 25
```

Comment:
May I suggest something like this:

```ruby
def records(object, options = {})
  defaults = {
    :rows_per_page => 25,
    :page_number => 1
  }
  hash_request = request
    .objects(object)
    .records
    .get(:params => defaults.merge(options))
  payload = payload_hash(hash_request)
  translate_payload(payload) { payload }
end
```

Maybe a refactor to try once there are tests :smile: